apps/ledger-live-mobile/src/screens/Swap/Form/Modal/index.tsx

```diff
@@ -55,6 +58,12 @@ export function Modal({
 
   const onError = useCallback(
     ({ error, swapId }) => {
+      track("error_message", {
+        ...sharedSwapTracking,
+        message: "drawer_error",
+        page: "Page Swap Drawer",
+        error: error?.name ?? "unknown",
```

Comment:
any specific case, why are we using nullish coalescing operator ? (only curiosity)